modules/ROOT/pages/api-gateway-encryption-mule4.adoc

```diff
@@ -0,0 +1,252 @@
+= Gateway Startup Encryption in Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Gateway startup encryption allows you to encrypt sensitive information required to configure the connection between your Mule application and Anypoint Platform.
```

Comment:
It also encrypts sensitive information stored by the Gateway such as policies and contracts.

Reply:
You are right, maybe changing it to something like this:

> Gateway startup encryption allows you to encrypt sensitive information stored by the runtime such as policies, contracts, and the required credentials to configure the connection between your Mule application and Anypoint Platform.

Reply:
Seems good to me.